sandbox/src/sandbox.rs

```diff
@@ -297,8 +302,24 @@ impl Sandbox {
         self.broadcast_to_addrs(msg, self.addresses.iter().skip(1));
     }
 
+    pub fn broadcast_nopanic<T: Message>(&self, msg: &T) -> Result<(), String> {
```

Comment:
It seems that spellcheck doesn't like "nopanic", neither do I. 😄 Maybe follow the `from`/`try_from` pattern (`try_broadcast`)?